alien-module/control.lua

```diff
@@ -36,7 +36,7 @@ function update_gui()
 		if player.gui.top.killcount == nil then
 			player.gui.top.add{type="label", name="killcount", caption="TEST"}
 		end
-		player.gui.top.killcount.caption = "Hypermodule level: " .. roundModuleLevel(3) .. " Killcount: " .. global.killcount
+		player.gui.top.killcount.caption = "异星超级插件等级: " .. roundModuleLevel(3) .. " 消灭异虫数量: " .. global.killcount
```

Comment:
Thanks for the translation but i think the english text would be preferable here